### PR TITLE
Update text style and use it more consistently

### PR DESCRIPTION
**Pull request by ghengeveld**
<details>
  <summary>📦 Published PR as canary version: <code>1.2.19--canary.217.dba9970.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromatic-com/storybook@1.2.19--canary.217.dba9970.0
  # or 
  yarn add @chromatic-com/storybook@1.2.19--canary.217.dba9970.0
  ```
</details>

### CODE REVIEW

File: src/components/Text.tsx [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
-  fontSize: small ? 12 : 14,
-  lineHeight: small ? "18px" : "20px",
-  textAlign: "center",
-  textWrap: "balance",
-}));
+export const Text = styled.div<{ center?: boolean; muted?: boolean; small?: boolean }>(
+  ({ center, small, theme }) => ({
+    display: "inline-block",
+    color: theme.color.defaultText,
+    fontSize: small ? theme.typography.size.s1 : theme.typography.size.s2 - 1,
```
Comment (MichaelArestad) [inline src/components/Text.tsx:7]: Storybook changed the base font sizes to 12px and 14px. Likewise, this should be 12px or 14px. There are a handful of exceptions like Button. Toolbar buttons should still be 12px, though.

File: src/components/BuildProgressBarInline.tsx [hunk 2]
```diff
@@ -14,23 +14,19 @@ const ProgressTextWrapper = styled(Text)({
...
   return (
-    <ProgressTextWrapper small as="div">
+    <ProgressTextWrapper center small>
       <ProgressTrack>
         {typeof localBuildProgress.buildProgressPercentage === "number" && (
           <ProgressBar style={{ width: `${localBuildProgress.buildProgressPercentage}%` }} />
         )}
       </ProgressTrack>
-      <StyledBuildProgressLabel localBuildProgress={localBuildProgress} />
+      <BuildProgressLabel center muted localBuildProgress={localBuildProgress} />
```
Comment (MichaelArestad) [inline src/components/BuildProgressBarInline.tsx:29]: I think this might need a `small` prop. Not entirely sure.

**Comment by MichaelArestad** [comment]
This is rad. Thanks for putting this together. I still need to finish reviewing. Will take a closer look after the meeting.

**Review by andrewortwein — APPROVED** [review]
👍🏻 Nice touch on the intro fragment

**Pull request merged**